Automattic/woocommerce-payments
# Remove "wcpay_current_session_id" filter from wc-payments-fraud-service

Fixes #3532 

#### Changes proposed in this Pull Request

Remove unused filter `wcpay_current_session_id` with undefined callback `get_session_id`

Psalm reported the issue about the unknown callback.
We can delete this line, and update the psalm baseline to remove the fixed issue as well as update the related unit test.

#### Testing instructions

- Open file `class-wc-payments-fraud-service.php`
- Run `./vendor/bin/psalm --show-info=true includes/class-wc-payments-fraud-service.php`
- You should not see an issue about "InvalidArgument" and "get_session_id"

Also made sure there were no occurrences of `wcpay_current_session_id` filter anywhere in our codebase (double-checked with opengrok and no occurrences found either)

-------------------


**Post merge**
